coderd/notifications_test.go

```diff
@@ -20,7 +20,6 @@ func createOpts(t *testing.T) *coderdtest.Options {
 	t.Helper()
 
 	dt := coderdtest.DeploymentValues(t)
-	dt.Experiments = []string{string(codersdk.ExperimentNotifications)}
```

Comment:
nit: maybe use comment instead or empty array with `FIXME`?

Reply:
With the feature not being experimental anymore and `Experiments` already having a zero value of an empty slice, I think removing this is appropriate.